README.md

```diff
@@ -44,6 +44,11 @@ opt
   * `args` - passed to `is_tuple_expired()` and `process_expired_tuple()` as additional context
   * `tuples_per_iteration` - number of tuples will be checked by one iteration
   * `full_scan_time` - time required for full index scan (in seconds)
+  * `on_iteration_error` - call function on error
+  * `on_iteration_start` - call function on iteration
+  * `on_iteration_complete` - call function on complete iteration
+  * `on_iteration_stop` - call function on end iteration cycle
```

Comment:
The terminology here is confusing (just like in the code itself). Those callbacks are invoked after the whole space traversal and the traverse / full scan is called 'iteration'. But `tuples_per_iteration` assumes 'iteration' relation to a chunk of tuples of given size. Don't know what to do with this naming...

Most logical would be name `tuples_per_iteration` as `chunk_size`. However, it is named so for years and renaming may lead to even more confusion. So, maybe, it worth to left the 'iteration' term to the chunk of tuples and name the new options as `on_traverse_*` or `on_full_scan_*`? The latter is consistent with `full_scan_time` that looks as the good property.

Reply:
Done.